apps/files/src/views/TemplatePicker.vue

```diff
@@ -179,6 +180,11 @@ export default defineComponent({
 
 			// Else, open the picker
 			this.opened = true
+
+			// Set initial focus to the empty template preview
+			this.$nextTick(() => {
+				this.$refs.emptyTemplatePreview?.$el.querySelector('input')?.focus()
```

Comment:
I think this is quite hacky as you assume there are specific DOM elements within an other component.
Instead it would be cleaner if you expose a method in `TemplatePreview` like `focus()` and just call:

```suggestion
				this.$refs.emptyTemplatePreview?.focus()
```

Reply:
Indeed makes sense, thanks for the helpful feedback